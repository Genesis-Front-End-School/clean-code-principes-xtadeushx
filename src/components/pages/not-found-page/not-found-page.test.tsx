import { screen, render } from '@testing-library/react';

import { NotFoundPage } from './not-found-page';

describe('NotFoundPage component', () => {
  it('should render successfully', () => {
    render(<NotFoundPage />);

    expect(screen.getByText('This page does not found')).toBeInTheDocument();
  });

  it('should match snapshot', () => {
    render(<NotFoundPage />);

    expect(screen.getByText('This page does not found')).toMatchSnapshot();
  });

  it('should have class name', () => {
    render(<NotFoundPage />);

    expect(screen.getByText('This page does not found')).toHaveClass(
      'not-found-page'
    );
  });
});
