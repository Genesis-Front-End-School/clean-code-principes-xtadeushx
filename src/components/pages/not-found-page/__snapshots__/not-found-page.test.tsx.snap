// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage component should match snapshot 1`] = `
<h2>
  <span>
    ðŸ˜•
  </span>
  <br />
  This page does not found
</h2>
`;
